warehouse/models/mart/gtfs_quality/_mart_gtfs_quality.yml

```diff
@@ -78,3 +78,63 @@ models:
           expression: "is_implemented"
     columns:
       - name: is_implemented
+
+  - name: fct_daily_reports_service
```

Comment:
move this table to `mart_gtfs`, just call it `fct_daily_scheduled_service_summary`